.changeset/brown-terms-confess.md

```diff
@@ -0,0 +1,5 @@
+---
+'@graphql-codegen/visitor-plugin-common': minor
```

Comment:
@kamilkisiela please change to `patch`. I know it's not ideal, but it's the best way to workaround conflicts later.

Reply:
Oh, ok. I'm testing it still so let's keep the pr open or as a draft